guides/release/configuring-ember/debugging.md

```diff
@@ -1,6 +1,9 @@
 Ember provides several configuration options that can help you debug problems
 with your application.
 
+## Ember Inspector
+The [Ember Inspector](https://github.com/emberjs/ember-inspector) is a browser extension that makes understanding and debugging your Ember.js application a snap. To learn more about it check out the [dedicated guide](../../ember-inspector/).
```

Comment:
Can we update the first sentence to not use the word "snap"? I think the idiom may be unfamiliar outside of North America.

```suggestion
The [Ember Inspector](https://github.com/emberjs/ember-inspector) is a browser extension that makes understanding and debugging your Ember.js application a snap. To learn more, check out the [dedicated guide](../../ember-inspector/).
```

Reply:
Absolutely, I took that sentence directly from the description of Ember Inspector on the Chrome web store. But I agree about the language. I will update to make it less idiomatic.